packages/core/objects/_main-wrapper.scss

```diff
@@ -17,48 +17,66 @@
 ///   </div>
 ////
 
-@mixin govuk-main-wrapper {
+@mixin _nhsuk-main-wrapper {
```

Comment:
I might have missed something but I noticed there is no `nhsuk-main-wrapper` mixin exposed, but the changelog says "All Sass mixins, functions and variables prefixed `govuk` are now prefixed `nhsuk`."

Does this need further clarification?